LICENSE

```diff
@@ -1,29 +0,0 @@
-BSD 3-Clause License
```

Comment:
Did you mean to delete the licence file?

Reply:
No, good catch. Thanks, updated.